core/modules/replica/Messenger.h

```diff
@@ -45,6 +45,7 @@ namespace qserv {
 namespace replica {
 
 // Forward declarations
+class Messenger;
```

Comment:
Why do you need this forward declaration, class itself is few lines below?

Reply:
👍